common/buildings/wc_cultural_tradepost.txt

```diff
@@ -0,0 +1,314 @@
+trade_post = {
+	
+	############CULTURAL#################
+	
+	### General Building
+	tp_road_tavern = {
+		potential = { in_major_trade_route_trigger = yes }
+		desc = tp_empty
+		trigger = {
+			FROM = { not_tradepost_culture_groups = yes }

```

Comment:
`not_tradepost_culture_groups`: such scripted trigger doesn't exist.

Reply:
I forgot to create it XD